src/test/java/com/google/cloud/pontem/TableInformationTest.java

```diff
@@ -340,4 +370,41 @@ public void testGetMapOfColumnNamesToSpannerTypesFromTableDdl_error2() {
 
     TableInformation tableInformation = new TableInformation(brokenDdl);
   }
+
+  @Test
+  public void testGetAvroTypeOfColumn() {
+    TableInformation tableInformation = new TableInformation(basicDdl1);
+    assertEquals(SchemaBuilder.builder().longType(), tableInformation.getAvroTypeOfColumn("SingerId"));
+  }
+
+  @Test(expected = RuntimeException.class)
+  public void testGetAvroTypeOfColumn_exception() {
+    TableInformation tableInformation = new TableInformation(basicDdl1);
+    tableInformation.getAvroTypeOfColumn("NoColumn");
+  }
+
+  @Test
+  public void testGetSpannerTypeOfColumn() {
+    TableInformation tableInformation = new TableInformation(basicDdl1);
+    assertEquals(Type.int64(), tableInformation.getSpannerTypeOfColumn("SingerId"));
+  }
+
+  @Test(expected = RuntimeException.class)
+  public void testGetSpannerTypeOfColumn_exception() {
+    TableInformation tableInformation = new TableInformation(basicDdl1);
+    tableInformation.getSpannerTypeOfColumn("NoColumn");
+  }
+
+  @Test
+  public void testIsColumnNullable() {
+    TableInformation tableInformation = new TableInformation(basicDdl1);
+    assertEquals(true, tableInformation.isColumnNullable("FirstName"));
+    assertEquals(false, tableInformation.isColumnNullable("SingerId"));
+  }
+
+  @Test(expected = RuntimeException.class)
+  public void testIsColumnNullable_error() {
+    TableInformation tableInformation = new TableInformation(basicDdl1);
+    tableInformation.isColumnNullable("NoSuchColumn");
```

Comment:
NoColumn so as to be consistent with the other names or NoSuchColumn for all?
Also why not extract these into constants?